### PR TITLE
added component NextDayButton and funcionality

### PR DESCRIPTION
**Pull request by lukaszgancarz98**

### CODE REVIEW

File: django_react_proj/users-fe/src/components/UserList/UserList.js [hunk 1]
```diff
@@ -19,6 +22,27 @@ const Actions = ({ user, resetState }) => {
     });
     resetState();
   };
+  const moveToPreviousDay = async () => {
+    const result = isSameDay(new Date(), new Date(user.startTime));
+    if (result === false) {
```
Comment (bakowroc) [inline django_react_proj/users-fe/src/components/UserList/UserList.js:27]: `!result`

File: django_react_proj/users-fe/src/components/UserList/UserList.js [hunk 2]
```diff
@@ -31,6 +55,10 @@ const Actions = ({ user, resetState }) => {
 
   return (
     <>
+      <PreviousDayButton moveToPreviousDay={moveToPreviousDay} />
+      <> </>
```
Comment (bakowroc) [inline django_react_proj/users-fe/src/components/UserList/UserList.js:59]: ?

**Review by bakowroc — APPROVED** [review]

File: django_react_proj/users-fe/src/components/UserList/UserList.js (hunk 1 above)
Comment (lukaszgancarz98) [reply django_react_proj/users-fe/src/components/UserList/UserList.js:27]: Value of result must be false to move task to previous day. If result is true (that means planned day of task is today) you can't move it to previous day.

File: django_react_proj/users-fe/src/components/PreviousDayButton/PreviousDayButton.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from 'antd/lib/button';
+import { ArrowLeftOutlined } from '@ant-design/icons';
+import 'antd-button-color/dist/css/style.css';
+
+const PreviousDayButton = ({ moveToPreviousDay }) => (
+  <>
```
[the remark this answers is not in the document]
Comment (lukaszgancarz98) [reply django_react_proj/users-fe/src/components/PreviousDayButton/PreviousDayButton.js:8]: Deleted.

File: django_react_proj/users-fe/src/components/NextDayButton/NextDayButton.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from 'antd/lib/button';
+import { ArrowRightOutlined } from '@ant-design/icons';
+import 'antd-button-color/dist/css/style.css';
+import { ToastContainer, toast } from 'react-toastify';
+
+const NextDayButton = ({ moveToNextDay }) => (
+  <>
```
[the remark this answers is not in the document]
Comment (lukaszgancarz98) [reply django_react_proj/users-fe/src/components/NextDayButton/NextDayButton.js:9]: Deleted.

File: django_react_proj/users-fe/src/components/UserList/UserList.js [hunk 5]
```diff
@@ -19,6 +22,27 @@ const Actions = ({ user, resetState }) => {
...
+  const moveToPreviousDay = async () => {
+    const result = isSameDay(new Date(), new Date(user.startTime));
+    if (result === false) {
+      await axios.put(`${API_URL + user.id}/`, {
+        startTime: addDays(new Date(user.startTime), -1),
+        endTime: addDays(new Date(user.endTime), -1),
+      });
+      resetState();
+      toast.info('You moved task to the previous day!');
+    } else if (result === true) {
```
[the remark this answers is not in the document]
Comment (lukaszgancarz98) [reply django_react_proj/users-fe/src/components/UserList/UserList.js:34]: When I remove it, eslint throws an error.

**Pull request merged**